cli/lsp/config.rs

```diff
@@ -720,4 +753,16 @@ mod tests {
       }
     );
   }
+
+  #[test]
+  fn test_empty_config() {
+    let mut config = Config::new();
+    config
+      .set_workspace_settings(json!({ "config": "" }))
+      .expect("could not update");
+    assert_eq!(
+      config.get_workspace_settings(),
```

Comment:
Could you add tests that check for empty strings in the values you changed?

Reply:
done